### PR TITLE
Refresh the experiment

### PR DESCRIPTION
**Pull request by ltratt**
This PR updates the experiment to use an upto date version of grmtools, remove MF, update the copyright, and update the instructions.

### CODE REVIEW

File: runner/java_parser/src/main.rs [hunk 1]
```diff
@@ -8,16 +8,16 @@ extern crate cfgrammar;
 use lrpar::{ParseRepair, LexParseError, Lexer};
 
 // Using `lrlex_mod!` brings the lexer for `calc.l` into scope.
-lrlex_mod!(java7_l);
-// Using `lrpar_mod!` brings the lexer for `calc.l` into scope.
-lrpar_mod!(java7_y);
+lrlex_mod!("java7.l");
+// Using `lrpar_mod!` brings the parser for `calc.l` into scope.
```
Comment (ptersilie) [inline runner/java_parser/src/main.rs]: /calc.l/calc.y

Comment (ltratt) [reply runner/java_parser/src/main.rs]: Oops. Fixed in https://github.com/softdevteam/error_recovery_experiment/pull/27/commits/2a0e29bdc420bcb5d899a5d7e471ec06d7c98c38

File: README.md [hunk 2]
```diff
@@ -2,8 +2,28 @@
...
-  2. Run the experiments
+  1. Install dependencies
+  2. Generate Blackbox data
+  3. Run the experiments
+
+
+## Install dependencies
+
+To run the experiments you need to install Python 2.7 and Rust. There are
+several ways to do install Rust, but [rustup](https://rustup.rs/) is generally
```
Comment (ptersilie) [inline README.md]: several ways to ~~do~~ install Rust

**Comment by ptersilie** [comment]
One more comment. Rest looks good.

File: README.md (hunk 2 above)
Comment (ltratt) [reply README.md]: Fixed in https://github.com/softdevteam/error_recovery_experiment/pull/27/commits/19467c1fcde635bfeaa8286bd2e976bb860f59c5

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Review by ptersilie — APPROVED** [review]

**Pull request merged**